src/pudl_archiver/utils.py

```diff
@@ -70,4 +70,5 @@ def add_to_archive_stable_hash(archive: zipfile.ZipFile, filename, data: bytes):
         # Set fixed date to enable hash comparisons between archives
         date_time=(1980, 1, 1, 0, 0, 0),
     )
+    info.compress_type = zipfile.ZIP_DEFLATED
```

Comment:
Huh! I guess the default value here is `zipfile.ZIP_STORED` which presumably causes some confusion in the decompression. TIL!

Also, surprising that compress_type can't be set at initialization time 🤷 

Might be useful to clarify these things in a comment.

```suggestion
    # default is ZIP_STORED, which means "uncompressed"; also this can't be set in the constructor as of 2024-02-09
    info.compress_type = zipfile.ZIP_DEFLATED
```